src/Pages/instructor/manage-class/class-row.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Link } from "react-router-dom";
+
+import { StyledLink, MainHeader, Banner } from "../../../reusable-components/reusableComponents";
+
+
+const StyledRow = styled.div`
+    display: flex;
+    width: 100%;
+    justify-content: space-evenly;
+
+    h6 {
+        width: 20%;
+        text-align: center;
+    }
+
+    .btns {
+        display: flex;
+        width: 20%;
+    }
+`
+const Button = styled(Link)`
+	background-color: ${pr => pr.theme.primaryOrange};
+	color: ${pr => pr.theme.charcoal};
+	font-size: 1.6rem;
+	border-radius: 5px;
+    font-weight: 500;
+    width: 100%;
+    margin: 5px;
+    padding: 2px 10px;
+	display: flex;
+	flex-flow: row nowrap;
+	justify-content: center;
+`
+
+
+const ClassRow = (props) => {
```

Comment:
this component is awesome, I love that you used a table. the layout looks really good overall, nice job on that front.